internal/retryhttp/client.go

```diff
@@ -78,6 +85,12 @@ func normalizeResponse(res *http.Response, err error) (*http.Response, error) {
 			return res, locale.WrapInputError(&UserNetworkError{429}, "err_user_network_toomany", "Request failed due to too many requests. {{.V0}}", solutionLocale)
 		}
 	}
+
+	var dnsError *net.DNSError
+	if errors.Is(err, dnsError) {
+		return res, locale.WrapError(&UserNetworkError{}, "err_user_network_dns", "Request failed due to DNS error. {{.V0}}", solutionLocale)
```

Comment:
```suggestion
		return res, locale.WrapError(&UserNetworkError{}, "err_user_network_dns", "Request failed due to DNS error: {{.V0}}. {{.V1}}", err.Error(), solutionLocale)
```

In this case I think we should actually include the error itself.